packages/core/src/Account.ts

```diff
@@ -221,6 +222,11 @@ export class Account extends TypedEventEmitter<Events> {
     return storeEngine.updateOrCreate(AUTH_TABLE_NAME, AUTH_COOKIE_KEY, entity);
   }
 
+  private async getE2EIStatus(): Promise<boolean> {
```

Comment:
**suggestion**: wouldn't `isE2EIFeatureEnabled` method name make more sense since `boolean` is what we return, not the actual status.

Reply:
I think it sounds like a boolean then. But it is a async function that we need to call.